src/scripts/add-robot-template-header.py

```diff
@@ -0,0 +1,28 @@
+import click
+import json
+import pandas as pd
+from io import StringIO
+
+@click.command()
+@click.argument('tsv_file', type=click.File('r'))
+@click.argument('json_file', type=click.File('r'))
+def add_robot_template_header(tsv_file, json_file):
+    # Load the JSON configuration
+    config = json.load(json_file)
+
+    # Read the TSV file into a DataFrame
+    df = pd.read_csv(tsv_file, sep='\t')
+
+    # Create a new row based on the JSON config
+    new_row = {col: config.get(col, '') for col in df.columns}
+
+    # Append the new row to the DataFrame
+    df = pd.concat([pd.DataFrame([new_row]), df], ignore_index=True)
+
+    # Output the modified DataFrame
+    output = StringIO()
+    df.to_csv(output, sep='\t', index=False)
+    click.echo(output.getvalue())
```

Comment:
I think this prints the entire df to the console, is that intended?

Reply:
Yes, this is how a lot of the unix tools operate. It save you the necessity of passing in an out file parameter and writing to it. Instead you simple redirect the output of this scripts to the terminal with the ">" symbol.

Reply:
gotcha. are there any size limits of the `output` we should be aware of or check for before this line? for example, how big can the `output` be and this work?

Reply:
Good point.. I can't think of anything - the main point of streaming to std out (other than supporting use in unix pipelines) would be that the memory consumption is minimised. But that is a bit beyond my expertise.. it's a frequently used pattern in our scripts (oak does it too) so I hope it's alright..

Reply:
I will close this now, because we have been doing it so far, but we will keep it in mind when size limits (Out of memory errors etc) are emerging.